tests/mu_define_enum_test.c

```diff
@@ -34,5 +42,13 @@ int run_mu_define_enum_tests(void)
 
     POOR_MANS_ASSERT(MU_ENUM_VALUE_COUNT(TEST4_ENUM_VALUES) == 3);
 
+    const char* TEST5_test5_b = MU_ENUM_TO_STRING(TEST5_ENUM, 42);
+    POOR_MANS_ASSERT(TEST5_test5_b != NULL);
+    POOR_MANS_ASSERT(strcmp("test5_b = 42", TEST5_test5_b) == 0);
+
+    const char* TEST5_unknown = MU_ENUM_TO_STRING(TEST5_ENUM, 41);
+    POOR_MANS_ASSERT(TEST5_unknown != NULL);
+    POOR_MANS_ASSERT(strcmp("NULL", TEST5_unknown) == 0);
+
     return result;
 }
```

Comment:
Consider having tests that do preserve the nature of the deleted tests. Fo example: there's no test for an enum declared with WITHOUT_INVALID macro.

Reply:
Added WITHOUT_INVALID tests

---
In reply to: [328361834](https://github.com/Azure/azure-macro-utils-c/pull/30#discussion_r328361834) [](ancestors = 328361834)